lib-util/src/main/java/org/stellar/anchor/util/NumberHelper.java

```diff
@@ -14,4 +16,17 @@ public static boolean isPositiveNumber(String str) {
       return false;
     }
   }
+
+  public static boolean hasProperSignificantDecimals(
```

Comment:
Should we remove `mixDecimals` for now? It's only used in tests, not in the main code. We can always add an extra parameter latert